src/App.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div id="app">
+    <h1 class="title">  Addons catalog </h1>
+    <table v-if="addons.length > 0">
+      <thead>
+        <tr>
+          <th>
+            Name
+          </th>
+          <th>
+            Version
+          </th>
+          <th>
+            Unstable
+          </th>
+          <th>
+            Compatibility
+          </th>
+          <th>
+            Download
+          </th>
+        </tr> 
+      </thead>
+      <tbody>
+        <tr v-for="addon in addons">
+          <td>
+            {{addon.name}}
+          </td> 
+          <td>
+            {{addon.version}}
+          </td> 
+          <td>
+            {{addon.unstable}}
+          </td>
+          <td>
+            {{addon.compatibility}}
+          </td>
+          <td> 
+            <a :href="addon.downloadUrl" ><i class="uiIconDownload"></i></a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+export default {
+  name: 'App',
+   created() {
+    this.fetchData();	
+  },
+  data() {
+    return {
+      addons: []
+    }
+  },
+  methods: {
+    fetchData() {
+      axios.get('http://storage.exoplatform.org/public/Addons/list.json').then(response => {
+        this.addons = response.data;
+      });
+    }
+  }
+}
+</script>
+
+<style>
+#app {
+  font-family: 'Avenir', Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+  margin-top: 60px;
+}
+body {
+  font-family: Helvetica Neue, Arial, sans-serif;
+  font-size: 14px;
+  color: #444;
+}
+
+table {
+  border: 2px solid #578dc9;
+  border-radius: 3px;
+  background-color: #fff;
+}
```

Comment:
We should center the table on the page, by adding the property "margin: auto;" on table.